index.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>myForm</title>
+    <link rel="stylesheet" href="./style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Raleway:wght@100;300;400&display=swap" rel="stylesheet">
+</head>
+<body>
+    <form action="https://httpbin.org/anything" method="POST">
+        <fieldset>
+            <legend>Your Donation Matters</legend>
+            <div class="text-field">
+                <label class="textfield-container" for="fname">Name:<br>
+                    <input type="name" id="fname" name="fname" value="Name">
```

Comment:
When we have many attributes like this, we can split them on multiple rows to increase readability:
```
<input
  type="name"
  id="fname"
  name="fname"
  value="Name">
```

Reply:
Also, test `placeholder="Name"` instead of `value="Name"`. That way the user doesn't have to erase what's already written ;)

Reply:
The type of the input should be `text` I think